cfme/infrastructure/pxe.py

```diff
@@ -533,25 +508,67 @@ def update(self, updates, cancel=False):
             msg = 'Customization Template "{}" was saved'.format(name)
         main_view.flash.assert_success_message(msg)
 
-    def delete(self, cancel=True):
+
+@attr.s
+class CustomizationTemplateCollection(BaseCollection):
+    """Collection class for CustomizationTemplate"""
+
+    ENTITY = CustomizationTemplate
+
+    def create(self, name, description, image_type, script_type, script_data, cancel=False):
+        """
+        Creates a Customization Template object
+
+        Args:
+            cancel (boolean): Whether to cancel out of the creation.  The cancel is done
+                after all the information present in the CT has been filled in the UI.
+            name: Name of CT
+            description:description: The description field of CT.
+            image_type: Image type of the CT.
+            script_data: Contains the script data.
+            script_type: It specifies the script_type of the script.
+        """
+
+        customization_templates = self.instantiate(name, description, script_data,
+                                                   image_type, script_type)
+        view = navigate_to(self, 'Add')
+        view.fill({'name': name,
+                   'description': description,
+                   'image_type': image_type,
+                   'type': script_type,
+                   'script': script_data})
+        main_view = self.create_view(PXECustomizationTemplatesView)
+        if cancel:
+            view.cancel.click()
+            msg = 'Add of new Customization Template was cancelled by the user'
+        else:
+            view.add.click()
+            msg = 'Customization Template "{}" was saved'.format(name)
```

Comment:
**Required**

This is failing in 59z tests, it looks like the description is in the flash message, not the name, because the screenshot shows a 16 character alpha-numeric.